common/src/test/java/tachyon/security/LoginUserTest.java

```diff
@@ -57,6 +57,43 @@ public void getSimpleLoginUserTest() throws Exception {
     Assert.assertFalse(loginUser.getName().isEmpty());
```

Comment:
Could you please also help to update this check to compare the user name, like below test? Thanks. 
It looks better to check the equality instead of just not empty. I did not realize this at that time.